lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -120,7 +126,7 @@ sub _before_render {
 
   my $format = $c->stash('format') || 'json';
   my $res
-    = $args->{exception} ? EXCEPTION() : !$has_spec ? NOT_FOUND() : $c->openapi->not_implemented;
+    = $args->{exception} ? EXCEPTION() : !$has_spec ? NOT_FOUND() : $c->openapi->not_implemented($args, $path);
```

Comment:
That is correct, $path can be derived. I left it there for brevity, but I guess it is better to be concise here :)
Can you answer my original question, about how you feel passing the $args to the exception handlers (NOT_IMPLEMENTED(), ...)